package/MDAnalysis/analysis/hole2/__init__.py

```diff
@@ -26,14 +26,14 @@
 
 due.cite(Doi("10.1016/S0006-3495(93)81293-1"),
          description="HOLE program",
-         path="MDAnalysis.analysis.hole",
+         path="MDAnalysis.analysis.hole2",
```

Comment:
Not super sure on these (nor on the addition of hol2 to MDAnalysis.analysis.__init__ would be worth specifically double checking here).

Reply:
What are you unsure about?

Reply:
Honestly it's my first time dealing with duecredit, I wasn't sure if this should be triggered on importing `MDAnalysis.analysis.hole2` or `MDAnalysis.analysis.hole2.hole`.